src/types.ts

```diff
@@ -79,6 +80,8 @@ export type AppState = {
   previousSelectedElementIds: { [id: string]: boolean };
   shouldCacheIgnoreZoom: boolean;
   showShortcutsDialog: boolean;
+  spreadsheet: Spreadsheet | null;
```

Comment:
what's this for?

Reply:
Need your help here.. I need the spreadsheet data inside the [`PasteChartDialog`] to render the actual previews.

Reply:
I see. We could also merge the props:

```
spreadsheetDialog: {
	shown: boolean;
	data: Spreadsheet | null;
}
```
or more type safe:
```
spreadsheetDialog: {
	shown: true;
	data: Spreadsheet;
} | {
	shown: false;
	data: null;
}
```
or simplified (but less semantic):
```
spreadsheetDialog: null | { data: Spreadsheet }
```

Reply:
Updated it.. also let's sort this list after we merge this please :)